src/main/java/com/maxmind/geoip2/record/AbstractNamedRecord.java

```diff
@@ -10,13 +11,17 @@
 /**
  * Abstract class for records with name maps.
  */
-public abstract class AbstractNamedRecord {
+public abstract class AbstractNamedRecord extends AbstractRecord {
 
     private final Map<String, String> names;
     private final Integer geoNameId;
     private final List<String> locales;
 
-    AbstractNamedRecord(Map<String, String> names, Integer geoNameId, List<String> locales) {
+    AbstractNamedRecord() {
```

Comment:
I don't know of anyone that requires them, but removing them would definitely be a breaking change under Semantic Versioning.